Frameworks/UIKit/UITextView.mm

```diff
@@ -759,7 +763,7 @@ - (CGSize)sizeThatFits:(CGSize)fitSize {
     centerRect.origin.y = 0;
     centerRect.size = fontExtent;
     // TODO(DH)
-    //EbrCenterTextInRectVertically(&centerRect, &fontExtent, _font);
+    // EbrCenterTextInRectVertically(&centerRect, &fontExtent, _font);
```

Comment:
@DHowett-MSFT is taking care of this

---
In reply to: [97468215](https://github.com/Microsoft/WinObjC/pull/1750#discussion_r97468215) [](ancestors = 97468215)